service/header/subscription.go

```diff
@@ -30,15 +30,14 @@ func newSubscription(topic *pubsub.Topic) (*subscription, error) {
 func (s *subscription) NextHeader(ctx context.Context) (*ExtendedHeader, error) {
 	msg, err := s.subscription.Next(ctx)
 	if err != nil {
-		log.Errorw("reading next message from subscription", "err", err.Error())
 		return nil, err
 	}
 	log.Debugw("received message", "topic", msg.Message.GetTopic(), "sender", msg.ReceivedFrom)
 
 	var header ExtendedHeader
 	err = header.UnmarshalBinary(msg.Data)
 	if err != nil {
-		log.Errorw("unmarshaling data from message", "err", err.Error())
+		log.Errorw("unmarshalling data from message", "err")
```

Comment:
Is there a reason to remove `err.Error()` from the string? Now the string is just "unmarshalling data from message err" all the time, no?

Reply:
Yeah, i don't understand why to remove `unmarshaling` as well as the error message?

Reply:
Oh, ok. Thank you for catching this. I wanted to only remove `.Error()` part.

Reply:
Fixed

Reply:
@adlerjohn, resolved and approved?